app/javascript/components/Developer/Tabs.tsx

```diff
@@ -17,31 +17,39 @@ export const Tabs = ({
   overlayTabpanelUID?: string;
   embedTabpanelUID?: string;
 }) => {
-  const selectTab = (evt: React.MouseEvent<HTMLAnchorElement>, tab: Tab) => {
+  const selectTab = (evt: React.MouseEvent<HTMLButtonElement>, tab: Tab) => {
     evt.preventDefault();
     setTab(tab);
   };
 
   return (
-    <UITabs>
-      <Tab
+    <div className="tab-buttons" role="tablist">
+      <Button
         onClick={(evt) => selectTab(evt, "overlay")}
-        isSelected={tab === "overlay"}
+        className={cx(tab === "overlay" ? "selected" : null)}
+        role="tab"
+        aria-selected={tab === "overlay"}
         aria-controls={overlayTabpanelUID}
       >
         <Icon name="stickies" />
         <div>
           <h4 className="tab-title">Modal Overlay</h4>
           <small>Pop up product information with a familiar and trusted buying experience.</small>
         </div>
-      </Tab>
-      <Tab onClick={(evt) => selectTab(evt, "embed")} isSelected={tab === "embed"} aria-controls={embedTabpanelUID}>
+      </Button>
+      <Button
+        onClick={(evt) => selectTab(evt, "embed")}
+        className={cx(tab === "embed" ? "selected" : null)}
```

Comment:
We should probably be using `classNames` from `$app/utils/classNames` now. It combines `cx` and `tailwind-merge` under the hood. Also it could be simplified:

```suggestion
        className={classNames({ selected: tab === "embed" })}
```

Reply:
Yeah, I'll update in bulk and remove `cx` entirely at some point